### ISSUE TITLE
dragging Part Feature into an Empty Body not working any more

### ISSUE THREAD

**Issue by wsteffe** [realthunder/FreeCAD#501]
Up to a few days ago I was able to drag a Part Feature (in example a Mirror) into an Empty Body.
May be I am doing it wrongly. I tried also to apply ctrl or ctrl+alt but not able to do it.
is anybody able to drag  Body (Mirror #1) into Body2 in the file [test.zip](https://github.com/realthunder/FreeCAD/files/9109161/test.zip) ?
![Screenshot](https://user-images.githubusercontent.com/12879321/178916939-9533e837-d03c-4261-9856-019ab89cc102.png)

**Comment by master-puller** [comment]
@wsteffe Check if you have changed any settings. When I drag the mirror into Body 2, I get a reference BaseFeature.
![image](https://user-images.githubusercontent.com/19511301/179031666-23e92942-67b6-46b5-aca6-19803d90a4c3.png)

**Comment by wsteffe** [comment]
@[master-puller](https://github.com/master-puller)[master-puller](https://github.com/master-puller)
Your image shows exactly what I wanted to have. 
The only setting I changed recently is "Allow cross coordinate links .." in Options/General/Objects.  Just reverted it but still not able to do what you did. 
When I am trying to drag Mirror into Body2, FC shows me a "no entry sign".
Have you synced recently with linkdaily ? Because I was able to do that before last sync/rebuild (3 days ago)
